Gem/Wallet/Views/WalletHeaderView.swift

```diff
@@ -20,7 +20,7 @@ struct WalletHeaderView: View {
     var action: HeaderButtonAction?
     
     var body: some View {
-        VStack(spacing: 12) {
+        VStack(spacing: Spacing.halfLarge) {
```

Comment:
```suggestion
        VStack(spacing: Spacing.large/2) {
```

let's use this instead to avoid extra names